app/experimenter/experiments/tests/factories/nimbus.py

```diff
@@ -246,8 +248,9 @@ def countries(self, create, extracted, **kwargs):
             self.countries.add(*extracted)
 
     @classmethod
-    def create_with_lifecycle(cls, lifecycle, **kwargs):
+    def create_with_lifecycle(cls, lifecycle, with_random_timespan=False, **kwargs):
```

Comment:
Oh yeah, and I made it an optional switch here because the random timing threw a lot of tests off